packages/math-input/src/components/tabbar/__tests__/tabbar_test.js

```diff
@@ -1,8 +1,7 @@
-import React from "react";
-import assert from "assert";
 import {mount} from "enzyme";
+import React from "react";
 
-import Tabbar from "../src/components/tabbar/tabbar";
+import Tabbar from "../tabbar.js";
```

Comment:
I hope we can change this to `import TabBar from "../tab-bar.js"` in the future.  `Tabbar` feels weird.  No action req'd for this PR.